Tests/Unit/MSFT_xFirewall.Tests.ps1

```diff
@@ -149,6 +149,7 @@ try
                 }
             }
             Context 'Ensure is Present and the Firewall is Absent' {
+                Mock Get-FirewallRule
```

Comment:
@PlagueHO - I had to add this to get the unit tests to pass on AppVoyer (no idea why it was working previously) can you validate the logic is still sound with this change and we're testing the right things?